Doc/reference/tohil_tcl_functions.rst

```diff
@@ -22,10 +22,14 @@ from Tcl interpreter, the following commands are available:
    If *-kwlist list* is specified, *list* contains key-value pairs
    that will be passed to the function as named parameters.
 
+   If *-nonevalue word* is specified, then this overrides the default sentinel
+   string.
+
    When you use tohil::call, Tohil converts all of your arguments
-   to Python Unicode, unless an argument is comprised of the special
-   sentinel `tohil::NONE`, in which case the Python "None" data type
-   is substituted in place of that argument.
+   to Python Unicode, unless an argument is comprised of  special
```

Comment:
i think you're missing a word before "special"?

Reply:
"a"

Reply:
ALso reminds me I need to update the documentation for my other ticket.